File: benchmarks/sqrt-1K/main.rs
use std::time::{SystemTime};

fn main() {
    let before = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();

    for i in 0..1000 {
        let _ = (i as f64).sqrt();
    }

    let after = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();
    println!("{}", after.as_nanos() - before.as_nanos());
}

File: benchmarks/sqrt-1K/main.go
package main

import "fmt"
import "time"
import "math"

func main() {
	var before = time.Now().UnixNano()

	for i := 0; i < 1000; i++ {
		math.Sqrt(float64(i))
	}

	var after = time.Now().UnixNano()

	fmt.Println(after - before)
}
